docs/includes/plugin_config.md

```diff
@@ -3,4 +3,4 @@ additional global and plugin configuration settings. These settings are used to
 modify metrics, tags, and field or create aliases and configure ordering, etc.
 See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 
-[CONFIGURATION.md]: ../../../docs/CONFIGURATION.md
+[CONFIGURATION.md]: ../../../docs/CONFIGURATION.md#plugins
```

Comment:
Making this change will modify all plug readmes, so you will need to run `make docs` along with this PR.

Reply:
I asked this on Slack, but didn’t get any response yet. I did it already but hadn’t pushed yet.